### PR TITLE
Feature/migrate symbols mui

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Migrate symbols mui into the symbols repo

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
+version: 2.1
+orbs:
+    gh: circleci/github-cli@1.0.3
 jobs:
     build:
         docker:
             - image: circleci/node:12.6-browsers
         steps:
             - checkout
-            - restore_cache:
```
Comment (joebochill) [inline .circleci/config.yml:8]: no more caching dependencies?

File: README.md [hunk 2]
```diff
@@ -1,85 +1,6 @@
...
->NOTE: If you will be using many of these symbols in your application, we recommend you use [@pxblue/symbols-mui](https://www.npmjs.com/package/@pxblue/symbols-mui) to simplify usage. This library makes more sense if you just need one or two symbols or if you want to reduce the size of your bundle.
-
-### Available Symbols
-Please see [Available Symbols](https://github.com/pxblue/symbols/blob/master/available_symbols.md) for a list of currently available symbols.
-
-# For Icon Creators
-Each symbol has its own folder in the /icons/symbols folder, which includes the clean SVG file .
-
-
+This is a repo that contains of one-line symbols for use in PX Blue applications as well as componentized symbols for use within a React application.
```
Comment (joebochill) [inline README.md:6]:
```suggestion
This is a repo that contains one-line symbols for use in PX Blue applications as well as componentized symbols for use within a React application.
```

Comment (joebochill) [reply README.md:6]: Links to the subfolder READMEs?

File: symbols-mui/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1 @@
+* @emclaug2 @joebochill @daileytj
```
Comment (joebochill) [inline symbols-mui/CODEOWNERS:1]: Are you allowed to have a codeowners file nested like this?

File: symbols-mui/scripts/generateSymbols.sh [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+echo -ne "${BLUE}Clearing existing symbols folder...${NC}"
+rm -rf symbols
+mkdir -p symbols
+echo -e "${BLUE}Done${NC}"
+
+echo -ne "Loading latest PX Blue icon package...${NC}"
+yarn upgrade @pxblue/symbols
+rm -rf node_modules/@pxblue/symbols/symbols.svg
+echo -e "${BLUE}Done${NC}"
```
Comment (joebochill) [inline symbols-mui/scripts/generateSymbols.sh:22]: These should be pulled from the /symbols folder, not from NPM (you'll also need to build them before you copy them over to make sure they are there and up to date)

File: symbols-mui/scripts/linkComponents.sh [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+echo -en "${BLUE}Copying build output into node_modules...${NC}";
+# cp -r ./symbols/package.json ./demos/showcase/node_modules/@pxblue/symbols-mui/package.json
+cp -r ./symbols/. ./demos/showcase/node_modules/@pxblue/symbols-mui
+# cp -r ./components/package.json ./demos/storybook/node_modules/@pxblue/symbols-mui/package.json
+# cp -r ./dist/. ./demos/storybook/node_modules/@pxblue/symbols-mui
+echo -e "${GREEN}Done${NC}"
+
+echo -en "\r\n${BLUE}Linking Components: ${NC}"
+if [ ! -f ./demos/showcase/node_modules/@pxblue/symbols-mui/package.json ]; then echo -e "${BRED}Not Linked${NC}" && exit 1; fi
+# if [ ! -f ./demos/storybook/node_modules/@pxblue/symbols-mui/package.json ]; then echo -e "${BRED}Not Linked${NC}" && exit 1; fi
```
Comment (joebochill) [inline symbols-mui/scripts/linkComponents.sh:31]: is this a duplicate of the line above? Can we get rid of these comments or do we need them?

File: symbols-mui/CODEOWNERS (hunk 3 above)
Comment (huayunh) [reply symbols-mui/CODEOWNERS:1]: has to be in the root, although you may technically change that `*` to say `symbols-mui/**` to specify code owners for a particular set of file.

File: .circleci/config.yml (hunk 1 above)
Comment (emclaug2) [reply .circleci/config.yml:8]: Yep this was dropped accidently and has been added back in.

File: symbols-mui/CODEOWNERS (hunk 3 above)
Comment (emclaug2) [reply symbols-mui/CODEOWNERS:1]: I've just removed the duplicated codeowners file and added Tom to the top-level CODEOWNERS

File: symbols-mui/scripts/linkComponents.sh (hunk 5 above)
Comment (emclaug2) [reply symbols-mui/scripts/linkComponents.sh:31]: I've removed all the comments, not sure how long they've been there for.

File: README.md [hunk 6]
```diff
@@ -1,85 +1,6 @@
...
->NOTE: If you will be using many of these symbols in your application, we recommend you use [@pxblue/symbols-mui](https://www.npmjs.com/package/@pxblue/symbols-mui) to simplify usage. This library makes more sense if you just need one or two symbols or if you want to reduce the size of your bundle.
-
-### Available Symbols
-Please see [Available Symbols](https://github.com/pxblue/symbols/blob/master/available_symbols.md) for a list of currently available symbols.
-
-# For Icon Creators
-Each symbol has its own folder in the /icons/symbols folder, which includes the clean SVG file .
-
-
+This is a repo that contains one-line symbols for use in PX Blue applications as well as componentized symbols for use within a React application.
```
Comment (joebochill) [inline README.md:6]: links to subfolder readmes?

**Review by joebochill — APPROVED** [review]
At a glance, this all looks good

**Comment by joebochill** [comment]
Did we want to consider renaming the /symbols/symbols folder to /symbols/symbols-svg?

**Comment by emclaug2** [comment]
> Did we want to consider renaming the /symbols/symbols folder to /symbols/symbols-svg?

I'm going to make an issue for this; I've tried to do a simple refactor but since the 'symbols' location is referenced in so many places, the refactor will probably require some time.  It's also more difficult because the output folder for the symbols-mui directory is called "symbols" as well, so this is confusing.

File: README.md (hunk 6 above)
Comment (emclaug2) [reply README.md]: Yep I've added some links now.

**Pull request merged**